apps/backoffice-v2/src/pages/Entity/components/DirectorsCallToAction/DirectorsCallToAction.tsx

```diff
@@ -0,0 +1,74 @@
+import { AnyObject } from '@ballerine/ui';
+import { FunctionComponent, useCallback, useMemo, useState } from 'react';
+import { CallToAction } from '../CallToAction/CallToAction';
+import {
+  ICallToActionDocumentOption,
+  ICallToActionDocumentSelection,
+  ICallToActionProps,
+} from '../CallToAction/interfaces';
+import { getRevisionReasonsForDocument } from './helpers';
+
+interface IDirectorsCallToActionProps extends ICallToActionProps {
+  documents: AnyObject[];
+  workflow: AnyObject;
+  onReset?: () => void;
+}
+
+export const DirectorsCallToAction: FunctionComponent<IDirectorsCallToActionProps> = ({
+  documents,
+  workflow,
+  onReset,
+  ...restProps
+}) => {
+  const [documentId, setDocumentId] = useState<string | null>(null);
+
+  const documentsOptions: ICallToActionDocumentOption[] = useMemo(
+    () =>
+      documents.map(({ category, type, id }) => ({
+        name: `${category as string} - ${type as string}`.replaceAll('_', ' '),
```

Comment:
Use conversion to title case from `string-ts` instead of `replaceAll`. The backoffice has an on-going task to have consistent text casing.